fetch.bs

```diff
@@ -2793,16 +2794,14 @@ optional <i>CORS flag</i> and <i>CORS-preflight flag</i>, run these steps:
  <li><p>Let <var>actualResponse</var> be null.
 
  <li>
-  <p>If <var>request</var>'s <a>skip-service-worker flag</a> is unset, then run these
+  <p>If <var>request</var>'s <a>service-workers mode</a> is not "<code>none</code>", then run these
   substeps:
 
   <ol>
    <li>
     <p>If <var>request</var>'s <a for=request>client</a> is null or
```

Comment:
I think that relying on service-workers mode is enough. At least, I thought the idea was that having a client no longer mattered.